flow/probes/ebpf.go

```diff
@@ -49,17 +46,11 @@ import (
 import "C"
 
 const (
-	BPF_ANY    = 0
-	ebpfUpdate = 2 * time.Second
+	BPF_ANY       = 0
+	FLOW_TABLE_SZ = 500000
+	ebpfUpdate    = 10 * time.Second
```

Comment:
under large stress, i.e. hundreds of thousands of different flows or more, 2 seconds become just illogical (empirically) since the agent needs much longer than that to go over the entire table.
Changing the number to 10 make much more sense, it's much cheeper in CPU since the agent can rest (unlike 2 where the scans happen all the time and kill the system). 
When the number of connections is low we don't lose anything since we will finally read all the flows.
As for a large number of connections, it seems that the average HTTP server for example handles around 30K connections per second, so since we have a table of 500K that we scan every 10 s, the average number of flows per second we can track is 50K which is bigger.